syncd/VendorSai.cpp

```diff
@@ -1300,6 +1354,52 @@ sai_status_t VendorSai::bulkRemove(
             object_statuses);
 }
 
+sai_status_t VendorSai::bulkRemove(
```

Comment:
@kcudnik I see this change is not relevant as per the PR title. Can this be kept outside this PR and added in another PR?

Reply:
read this comment: https://github.com/sonic-net/sonic-sairedis/pull/1430#issuecomment-2405155952, this is 1st pr: https://github.com/sonic-net/sonic-sairedis/pull/1431